buildsystem/buildsystem/zstackbuild.py

```
@@ -1,15 +1,15 @@
+from __future__ import absolute_import
```

Ruff (0.15.4)

[error] The file is executable but no shebang is present

(EXE002)